decision_making.md

```diff
@@ -0,0 +1,45 @@
+
+Depending on the governing body, decisions may be proposed by members of the decision making body, or by a broader group of stakeholders/contributors.
+
+**Informal consensus seeking.** Decision making starts with informal consensus seeking. The goal of this discussion phase is to refine the proposal, consider alternatives, weigh trade-offs, and attempt to find informal consensus. The legitimacy of the consensus seeking process is predicated on all stakeholders having their voices heard; teams must be proactive in providing opportunity for all relevant stakeholders to provide input. If the decision making body arrives at informal consensus, they may immediately move to document and enact the decision. This is the consensus seeking phase.
+
+**Calling a vote.** When the discussion matures during the consensus seeking phase, any member of the decision making body can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current form and notify the entire decision making body. After the proposal is seconded by another member of the decision making body, members have 7 days to vote. The governing body may consider longer voting periods as necessary for special circumstances, or _shorter periods only if all voting members are present_. The result will be determined by a simple majority for binary decisions and ranked choice for multiclass decisions (one among many, or several among many). The sponsor may update the proposal at any point during the voting period, in which case, the voting period will be reset.
+
+**Voter participation and quorum.** The various Jupyter decision making bodies often have vast scopes and members are often busy. As a result, there are typically a subset of members who have the time, interest, and expertise to dive into the details of a given decision. We believe that open, transparent, and time-bound voting with no quorum will enable these ad-hoc, informal subgroups to form spontaneously on a per-decision basis, in a manner that supports our goals of responsiveness and engagement.
+
+All members of a decision making body are required to participate in at least 2/3 of formal votes of that decision making body per calendar year (teams can decide how to account for the specifics of this in low-activity projects, etc.). Members that have not met the 2/3 vote participation threshold for a year will automatically be asked to step down at the end of that year. Those individuals remain eligible to rejoin the decision making body in the future as they become available to participate at the required level. The quorum for all formal votes will be 50% and a "blank" option will always be included, with "blank" option counting towards the quorum but not included in totals for calculating results.
```

Comment:
I would be explicit on the decision making bodies, including Board of Directors, SSC, and Subprojects (perhaps).

Reply:
Good points folks, thanks! We've added a section at the bottom clarifying these transition points. We made it a separate one so that it can be more easily removed later once the transition is complete, without having to do too much surgery inside the doc.